client/src/components/order/shoppingCart/shoppingCart.js

```diff
@@ -112,8 +113,8 @@ class ShoppingCart extends Component {
                                         disabled={this.getSubmitDisabled()}
                                         onClick={this.submitOrder}
                                     >
-                                        Beställ - {this.calculateTotalPrice()}
-                                        kr
+                                        Beställ -{" "}
+                                        {this.props.calculateTotalPrice} kr
```

Comment:
Weird js insertion here for the space.
Replace with a normal space and let Prettier autoformat :)
```suggestion
                                        Beställ - 
                                        {this.props.calculateTotalPrice} kr
```